docker-compose.yml

```diff
@@ -82,8 +92,10 @@ services:
       - '${REPO_PATH_SERVER}:/var/www/html'
       - '${REPO_PATH_SERVER}/apps-extra:/var/www/html/apps-extra'
       - '${ADDITIONAL_APPS_PATH:-./data/apps-extra}:/var/www/html/apps-shared'
-      - data:/var/www/html/data
-      - config:/var/www/html/config
+      #- data:/var/www/html/data
+      - '${SERVER_DATA_PATH:-data}:/var/www/html/data'
+      - '${SERVER_CONFIG_PATH:-config}:/var/www/html/config'
```

Comment:
I would prefer to keep those as docker volumes which can get automatically cleaned up when recreating the environment. Can you elaborate on why this would be needed?

Reply:
Yes this functionality is preserved as the default one. If ones want to locally access the files under data or config (which is needed for some scenarios) she can edit the .env and modify the SERVER_DATA_PATH  SERVER_CONFIG_PATH vars. This optional again for people knowing what thy want!
```
#SERVER_DATA_PATH=$BASE/workspace/server-data
#SERVER_CONFIG_PATH=$BASE/workspace/server-config
```
Some usage examples. 
When needing to easily modify server config you need the server-config outside.
When needing to mangle with my use case of testing server side symlinks it was more convenient to also have the server's data dir outside.

Reply:
>  When needing to easily modify server config you need the server-config outside.

(considering you are on latest master of this repo) https://juliushaertl.github.io/nextcloud-docker-dev/customizing/config/#nextcloud-configphp would cover that already

>  When needing to mangle with my use case of testing server side symlinks it was more convenient to also have the server's data dir outside.

I see, though I'd say this a quite special use case I'm undecided if that shall be generally a thing. Might then be unexpected that it only works for the main nextcloud container, not for others. Was there any reason to not just use a shell in the container using `docker-compose exec nextcloud bash` ?